test/groovy/AbstractPiperUnitTest.groovy

```diff
@@ -0,0 +1,28 @@
+import com.lesfurets.jenkins.unit.BasePipelineTest
+import org.junit.Before
+import org.yaml.snakeyaml.Yaml
```

Comment:
Could not be resolved for me. Use instead:
`org.jenkinsci.plugins.pipeline.utility.steps.shaded.org.yaml.snakeyaml.Yaml`

Reply:
This was solved by commit a92355c: "Update pom.xml"